project/LinkerdBuild.scala

```diff
@@ -360,6 +361,31 @@ object LinkerdBuild extends Base {
          |  done
          |fi
          |export MALLOC_ARENA_MAX=2
+         |
+         |# Configure GC logging directory
+         |if [ -z "$GC_LOG" ]; then
+         |  GC_LOG="/var/log/namerd"
+         |fi
+         |
+         |mkdir -p "$GC_LOG" && touch "$GC_LOG/gc.log"
```

Comment:
Wanted to make a note about the the line `touch "$GC_LOG/gc.log"`. It looks like even `mkdir -p $GC_LOG` will have an exit status of 0 if the directory already exists and has strict permissions. We actually need to do an additional check to see if our process can create a file in the directory since that is what the JVM will do when its writing logs to that directory.